src/CollectorMapper/MethodCallCollectorMapper.php

```diff
@@ -60,8 +60,10 @@ public function mapToLocalAndExternal(
      * @param array<string, mixed[]> $staticCallReferencesByFile
      * @return string[]
      */
-    private function mergeAndFlatten(array $methodCallReferencesByFile, array $staticCallReferencesByFile): array
-    {
+    private function mergeAndFlatten(
```

Comment:
all this whitespace changes were triggered by `composer fix-cs`

Reply:
Ah, that looks like new rule in PHP CS Fixer then :) I'll check it, thanks

Reply:
Interesting, I've just run `composer fix-cs` and there are no changes :thinking:  

How is your local setup different to the CI?

Reply:
I am running 

```
$ php -v
PHP 8.1.2 (cli) (built: Jan 19 2022 10:13:52) (NTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies
    with blackfire v1.76.0~win-x64-non_zts81, https://blackfire.io, by Blackfire

```

on windows11x64 (latest patches)

`composer fix-cs` leads to this diff: https://gist.github.com/staabm/aac750bf064f46868d0c52241bd3d1d8

Reply:
I see. How does it change if you exclude one of these 2 rules?

```bash
 * Symplify\CodingStandard\Fixer\LineLength\DocBlockLineLengthFixer
 * Symplify\CodingStandard\Fixer\LineLength\LineLengthFixer
```

I suspect the `DocBlockLineLengthFixer` :)

Reply:
Thanks :+1:

Reply:
@samsonasik Hi Abdul, I think we'll need a new test case with custom line-endings to fix it. 
Could you look at it when you're back? No rush

Reply:
I will try